packages/components/src/components/accordion/component.tsx

```diff
@@ -50,7 +50,6 @@ export class KolAccordion implements API {
 					<kol-heading-wc _label="" _level={this.state._level}>
 						<kol-button-wc
 							ref={this.catchRef}
-							slot="expert"
```

Comment:
Revert von dieser Änderung von heute morgen: https://github.com/public-ui/kolibri/pull/5535/files#diff-63aa29018079450cb476ce9b09bd26a9474766ef1bfc5b1e10a8d697786fb2f0L46

Ist in der Form ein Bug, weil kol-heading-wc nur einen default, keinen expert slot hat. Ich würde vorschlagen, dass in einem neuen Ticket zu refactorn.